### PR TITLE
Fix templates -> examples and cover all our examples

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Bugfix
- Documentation

### Description

The docs refer to our examples as to templates in the chapter 2.4. It also lists only twwo example files while we currently have 5 of them.

### CODE REVIEW

File: documentation/assemblies/assembly-kafka-cluster.adoc [hunk 1]
```diff
@@ -20,11 +20,15 @@ acquired using a `PersistentVolumeClaim` to make it independent of the actual ty
 ifdef::Kubernetes[HostPath volumes on Minikube or]
 Amazon EBS volumes in Amazon AWS deployments without any changes in the YAML files. The `PersistentVolumeClaim` can use a `StorageClass` to trigger automatic volume provisioning.
 
-{ProductName} includes two templates for deploying a Kafka cluster:
+{ProductName} includes several examples for deploying a Kafka cluster.
 
-* `kafka-ephemeral.yaml` deploys an ephemeral cluster, named `my-cluster` by default.
-* `kafka-persistent.yaml` deploys a persistent cluster, named `my-cluster` by default.
+* `kafka-persistent.yaml` deploys a persistent cluster with 3 Zookeeper and 3 Kafka nodes.
```
Comment (laidan6000) [inline documentation/assemblies/assembly-kafka-cluster.adoc]:
```suggestion
* `kafka-persistent.yaml` deploys a persistent cluster with three ZooKeeper and three Kafka nodes.
```

File: documentation/assemblies/assembly-kafka-cluster.adoc [hunk 2]
```diff
@@ -20,11 +20,15 @@ acquired using a `PersistentVolumeClaim` to make it independent of the actual ty
...
+* `kafka-jbod.yaml` deploys a persistent cluster with 3 Zookeeper and 3 Kafka nodes (each using multiple persistent volumes).
```
Comment (laidan6000) [inline documentation/assemblies/assembly-kafka-cluster.adoc]:
```suggestion
* `kafka-jbod.yaml` deploys a persistent cluster with three ZooKeeper and three Kafka nodes (each using multiple persistent volumes).
```

File: documentation/assemblies/assembly-kafka-cluster.adoc [hunk 3]
```diff
@@ -20,11 +20,15 @@ acquired using a `PersistentVolumeClaim` to make it independent of the actual ty
...
+* `kafka-persistent-single.yaml` deploys a persistent cluster with 1 Zookeeper and 1 Kafka nodes.
```
Comment (laidan6000) [inline documentation/assemblies/assembly-kafka-cluster.adoc]:
```suggestion
* `kafka-persistent-single.yaml` deploys a persistent cluster with a single ZooKeeper node and a single Kafka node.
```

File: documentation/assemblies/assembly-kafka-cluster.adoc [hunk 4]
```diff
@@ -20,11 +20,15 @@ acquired using a `PersistentVolumeClaim` to make it independent of the actual ty
...
+* `kafka-ephemeral.yaml` deploys an ephemeral cluster with 3 Zookeeper and 3 Kafka nodes.
```
Comment (laidan6000) [inline documentation/assemblies/assembly-kafka-cluster.adoc]:
```suggestion
* `kafka-ephemeral.yaml` deploys an ephemeral cluster with three ZooKeeper and three Kafka nodes.
```

File: documentation/assemblies/assembly-kafka-cluster.adoc [hunk 5]
```diff
@@ -20,11 +20,15 @@ acquired using a `PersistentVolumeClaim` to make it independent of the actual ty
...
+* `kafka-ephemeral-single.yaml` deploys an ephemeral cluster with 3 Zookeeper and 1 Kafka nodes.
```
Comment (laidan6000) [inline documentation/assemblies/assembly-kafka-cluster.adoc]:
```suggestion
* `kafka-ephemeral-single.yaml` deploys an ephemeral cluster with three ZooKeeper nodes and a single Kafka node.
```

**Comment by scholzj** [comment]
@laidan6000 The changes should be in now. Thanks.

**Pull request merged**